lib/idl/idl_checkers.es6.js

```diff
@@ -65,5 +65,21 @@ module.exports = [
     logger.info('Checking top-level member names');
 
     eachKey(logger, left.members || [], right.members || [], 'name');
-  }
+  },
+  function rightMissingConstructor(logger, left, right) {
+    function isCtor(attr) {
+      return attr.name === 'Constructor';
+    }
+    function hasConstructor(idl) {
+      return idl.attrs && idl.attrs.some(isCtor);
+    }
+    const chk = check.bind(this, logger);
+    if (hasConstructor(left) && !hasConstructor(right)) {
+      const ctors = JSON.stringify(left.attrs.filter(isCtor), null, 2);
+      logger.error(
```

Comment:
I thought left is blink and right is spec. Shouldn't we be checking if left is missing a constructor?